queries.sql

```diff
@@ -8,3 +8,12 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = 't';
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- new queries
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT COUNT(neutered), neutered FROM animals GROUP BY neutered;
+SELECT MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight, species FROM animals GROUP BY species;
+SELECT AVG(escape_attempts) AS avg_escape_attempts, species FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species
```

Comment:
- [ ] Kindly add your transaction queries into your queries.sql, I can see you did it from the screenshots you put in your PR
```
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; 

   SELECT species from animals; 
   ROLLBACK;

   SELECT species from animals;
```

Reply:
Alright, thanks for the feedback